Build/SaveLoadScreen.cc

```diff
@@ -1580,8 +1830,16 @@ static void SaveGameToSlotNum(void)
 	//render the buttons
 	MarkButtonsDirty( );
 	RenderButtons();
-
-	if( !SaveGame( gbSelectedSaveLocation, gzGameDescTextField ) )
+  
+  // If we are selecting the Dead is Dead Savegame slot, only remember the slot, do not save
+  // Also set the INTRO_SCREEN as previous options screen. This is a hack to get the game started
+  if (guiPreviousOptionScreen == GAME_INIT_OPTIONS_SCREEN)
+  {
+    guiPreviousOptionScreen = INTRO_SCREEN;
+    gGameSettings.bLastSavedGameSlot = (gbSelectedSaveLocation + NUM_SAVE_GAMES);
+    wcscpy(gGameSettings.sCurrentSavedGameName, gzGameDescTextField);
```

Comment:
oops,
but yes. both gGameSettings.sCurrentSavedGameName and gzGameDescTextField are initialized with SIZE_OF_SAVE_GAME_DESC